src/test/java/it/pagopa/transactions/commands/handlers/v2/TransactionsActivationProjectionHandlerTest.java

```diff
@@ -24,16 +24,26 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
 
 @ExtendWith(MockitoExtension.class)
 class TransactionsActivationProjectionHandlerTest {
 
-    @InjectMocks
     private TransactionsActivationProjectionHandler handler;
 
     @Mock
     TransactionsViewRepository transactionsViewRepository;
 
+    @BeforeEach
+    void setUp() {
+        handler = new TransactionsActivationProjectionHandler(
+                transactionsViewRepository,
+                true
+        );
+    }
+
```

Comment:
avoid initialize this bean for each test, initialize it once (as done for AuthorizationProjectionHandlerTest for example)